Content.Shared/Humanoid/Markings/MarkingCategories.cs

```diff
@@ -15,6 +15,7 @@ public enum MarkingCategories : byte
         Snout,
         SnoutCover,
         Chest,
+        NeckFluff,
```

Comment:
Also add TheDen comment

```suggestion
        NeckFluff, // TheDen - Ovinia, for fluff on necks
```